src/content/ccip/tutorials/usdc.mdx

```diff
@@ -0,0 +1,155 @@
+---
+section: ccip
+date: Last Modified
+title: "Transfer USDC and Data Across Chains"
+---
+
+import { Aside, ClickToZoom, CodeSample, CopyText } from "@components"
+
+USDC is a digital dollar backed 100% and is always redeemable 1:1 for US dollars. The [stablecoin](https://chain.link/education-hub/stablecoins) is issued by [Circle](https://www.circle.com/en/usdc) on multiple blockchain platforms.
+
+This guide will first explain how Chainlink CCIP enables native USDC transfers under the hood by leveraging [Circle's Cross-Chain Transfer Protocol (CCTP)](https://www.circle.com/en/cross-chain-transfer-protocol). Then, you will learn how to use Chainlink CCIP to transfer USDC and arbitrary data from a smart contract on _Avalanche Fuji_ to a smart contract on _Polygon Mumbai_.
+**Note**: You are not limited to programmable token transfers but can use CCIP to transfer USDC tokens without data. Check the [Mainnets](/ccip/supported-networks/mainnet) and [Testnets](/ccip/supported-networks/testnet) configuration pages to learn on which blockchains CCIP supports USDC transfers.
```

Comment:
```suggestion
**Note**: In addition to programmable token transfers, you can also use CCIP to transfer USDC tokens without data. Check the [Mainnets](/ccip/supported-networks/mainnet) and [Testnets](/ccip/supported-networks/testnet) configuration pages to learn on which blockchains CCIP supports USDC transfers.
```

Reply:
I also considered adding that programmable token transfers = tokens + data, since that association might not be immediately apparent to newer CCIP users. But if this guide is meant for more advanced users who would already know that, disregard